src/app/containers/StoryPromo/Figure.test.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { render } from 'react-testing-library';
+import { RequestContextProvider } from '../../contexts/RequestContext';
+import {
+  shouldShallowMatchSnapshot,
+  isNull,
+} from '../../helpers/tests/testHelpers';
+import StoryPromoFigure from './Figure';
+
+const imageProps = {
+  path: '/foobar.png',
+  altText: 'Alt text for an image',
+  height: 120,
+  width: 340,
+  copyrightHolder: 'Image provider',
+};
+
+describe('StoryPromo Figure Container', () => {
+  shouldShallowMatchSnapshot(
```

Comment:
This snapshot test is redundant for a Container unit test, when we have all the assertion-based tests below.

Reply:
I added them as the assertion tests test what is there, not what isn't 😆 Mainly to known if something is added by mistake. I see it as white noice, until theres a diff in a new PR that highlights an issue, They dont have much value the first time round imo

Reply:
Agreed that they can be useful to highlight cases where there are new items or attributes pulled in.